client/api_object.go

```diff
@@ -116,6 +114,19 @@ func (c *Client) CreateObject(ctx context.Context, bucketName, objectName string
 		return "", err
 	}
 
+	// This code block checks for unsupported or potentially risky formats in object names.
+	// The checks are essential for ensuring the security and compatibility of the object names within the system.
+	// 1. ".." in object names: Checked to prevent path traversal attacks which might access directories outside the intended scope.
+	// 2. Object name being "/": The root directory should not be used as an object name due to potential security risks and ambiguity.
+	// 3. "\\" in object names: Backslashes are checked because they are often not supported in UNIX-like file systems and can cause issues in path parsing.
+	// 4. SQL Injection patterns in object names: Ensures that the object name does not contain patterns that could be used for SQL injection attacks, maintaining the integrity of the database.
+	if strings.Contains(objectName, "..") ||
+		objectName == "/" ||
+		strings.Contains(objectName, "\\") ||
+		utils.IsSQLInjection(objectName) {
+		return "", fmt.Errorf("fail to check object name:%s", objectName)
+	}
```

Comment:
Could you wrap these format check codes into a single function?

Reply:
Please also move the object name rules into the CheckObjectName function, not comment here.

Reply:
CheckObjectName already has the same comments, so i just delete it